problem_builder/public/js/answer.js

```diff
@@ -35,9 +35,11 @@ function AnswerBlock(runtime, element) {
             if (result.status) {
                 if (result.status === "correct") {
                     checkmark.addClass('checkmark-correct icon-ok fa-check');
+                    checkmark.attr('aria-label', checkmark.data('label_correct'))
```

Comment:
Ahh... switching back and forth between JS and python does this 😄 
Fixed with b7ceb8c.